vdom/core.py

```diff
@@ -20,20 +33,25 @@ def _repr_mimebundle_(self, include, exclude, **kwargs):
                 'application/vdom.v1+json': toJSON(self.obj)
         }
 
+
 def createElement(tagName):
+    """Take an HTML tag and create an element"""
```

Comment:
I picked the worst name here, since these are more like `createClass`. I need to follow up with some changes here that will make this match the `createElement` APIs a bit better. @mpacer and I did some hacking with this yesterday, I'll make a PR after merging this one.

Reply:
Excellent. Enjoy day 3 of class with @mpacer.